dist/srpm/README.md

```diff
@@ -0,0 +1,19 @@
+This directory includes files necessary to build an SRPM as part of the normal
+`meson compile` step.
+
+# Usage
+
+1. Set the `build_srpm` build option to `True` when setting up the project:
+   `meson setup -Dbuild_srpm=True builddir`
+2. Compile the `srpm` target: `meson compile srpm -C builddir`
```

Comment:
This is great. We will be able to create scratch build of `yggdrasil` :+1:

Reply:
Yes. Creating the SRPM as a distinctly separate step was intentional. In a way, a SRPM is an alternate "source distribution" format, so it fits within the responsibility of meson nicely.